lib/task-data/schemas/install-coreos.json

```diff
@@ -40,10 +33,16 @@
             "$ref": "types-installos.json#/definitions/UserPassword"
         },
         "ignitionScriptUri": {
-            "$ref": "#/definitions/IgnitionScriptUri"
+            "$ref": "types-installos.json#/definitions/IgnitionScriptUri"
         },
         "progressMilestones": {
             "$ref": "types-installos.json#/definitions/ProgressMilestones"
+        },
+        "vaultToken": {
+            "$ref": "types-installos.json#/definitions/VaultToken"
```

Comment:
@yyscamper, this patchset only adds support for CoreOS, however I believe it is useful for other distros to be able to pull installation configuration (e.g. kickstart/preseed/autoyast) from Vault also. If you'd prefer me to move the options to install-coreos.json until there's another use case that leverages Vault, I can do so. Just please let me know how you'd like me to proceed. Thank you!